build.gradle

```diff
@@ -64,10 +64,10 @@ ext {
     compileSdkVersion = 29
     minSdkVersion = 16
     targetSdkVersion = 29
-    versionCode = 19
+    versionCode = 21
 
     buildToolsVersion = "29.0.2"
-    versionName = "3.2.0-alpha02"
+    versionName = "3.3.0-alpha02"
```

Comment:
The version name here and in gradle.properties are inconsistent.

Reply:
I'm updating those on purpose @nathanstasin

Reply:
Our pipelines for versioning aren't updated on this repo. It means there's a little bit of a mismatch in the between versioning on MavenCentral vs the tags here.